src/main/java/com/matt/forgehax/asm/utils/remapping/MCPMappingLoader.java

```diff
@@ -22,7 +23,7 @@
   private final String baseSrgDir = "{mc_ver}";
   private final String baseMappingDir = "{mc_ver}/{channel}_{map_ver}";
   private final String baseMappingUrl =
-    "http://export.mcpbot.bspk.rs/mcp_{channel}/{map_ver}-{mc_ver}/mcp_{channel}-{map_ver}-{mc_ver}.zip";
+    "https://github.com/ModCoderPack/MCPMappingsArchive/raw/master/mcp_snapshot/20171003-1.12/mcp_snapshot-20171003-1.12.zip";
```

Comment:
I know that it doesn't matter anymore, but this could be changed to be `https://github.com/ModCoderPack/MCPMappingsArchive/raw/master/mcp_{channel}/{map_ver}-{mc_ver}/mcp_{channel}-{map_ver}-{mc_ver}.zip`

Reply:
First: It does matter. 
Second: Done.